Samples/TdLib.Samples.GetChats/Program.cs

```diff
@@ -1,16 +1,21 @@
 ﻿using System;
+using System.IO;
 using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
-using TdLib.Bindings;
```

Comment:
Please revert this back, and you won't need the new extensions in `LegacyCompat`.

Reply:
Okay. I'm prepared to discuss this.
If I revert this code, then sample would not be valid for `TDSharp=1.8.1`. If I use `using TDLib.Bindings;` then I need compat.

That's sample which working with `TDSharp=1.8.1` https://github.com/kant2002/tdsharp-samples

Reply:
I don't think we need code valid for 1.8.1 in the `master` branch.

Moreover: we don't need code in `LegacyCompat` which will "restore" the compatibility with your sample which is in `master` but tries to use 1.8.1 APIs for whatever reason.

Just drop the legacy stuff.

Reply:
Last question. What's proper location for samples which works on the latest released version?

Reply:
I'd guess they should just be in the repository.

If someone wants a sample which works for an older release, they're free to checkout the older release tag and take the sample from there.

Reply:
I think I'm done with that sample